server/lib/app_manager.coffee

```diff
@@ -59,6 +63,14 @@ class AppManager
                     state: data.app.state
                 callback null, data.app.port
 
+    # readfile to start a static app
+    startStaticApp: (appName, url, path, callback) ->
+        logger.info "Starting static app #{appName}"
+        url += 'index.html' if url is '/' or url is '/public/'
+        fs.readFile path + url, 'utf8', (err, content) ->
```

Comment:
Please with coffeescript don't use `+` to build your strings. It's better if you use `"#{var1}#{var2}"`.